src/addon.cpp

```diff
@@ -26,49 +27,58 @@ class SHA3Hash: public ObjectWrap {
 
 	static void
 	Initialize(Handle<Object> target) {
-		HandleScope scope;
-		Local<FunctionTemplate> t = FunctionTemplate::New(New);
+		NanScope();
```

Comment:
@vladlosev It's a macro I think. The example code (throughout that entire file) doesn't ever do that: https://github.com/rvagg/nan#api_nan_scope